MDC2020/gen_OffSpill.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/bash
+#
+# this script requires mu2etools, mu2efiletools and dhtools be setup
+# It also requires the SimEfficiencies for the beam campaign be entered in the database
+# $1 is the name of the primary (ie CeEndpoint, etc).
+# $2 is the dataset description
+# $3 is the campaign version of the input file.
+# $4 is the campaign version of the output and primary file.
+# $5 is the number of input collections to merge (merge factor)
+rm template.fcl
+samweb list-file-locations --schema=root --defname="dts.mu2e.$1.$2$4.art"  | cut -f1 > $1$4.txt
```

Comment:
shouldn't the dataset be `$2$3` (and the name of the txt file `$1$3.txt`)

Reply:
In mixing we allow the primary to have a different version than the mixin files, but we assume the primary was produced with the same version as the current production.  We can make the same assumption here and then we need only 1 version variable.  Since primaries are cheap to produce I think it's reasonable.

Reply:
Actually this logic might not be right: cosmics are not a cheap primary, and if we ever want to re-digitize a cosmic primary with a later version of Offline this script wouldn't work.  I'll just fix it according to your suggestion.